main.py

```diff
@@ -431,6 +433,32 @@ async def list_result_directory() -> List[int]:
     return rid_list
 
 
+class ResultFileType(str, Enum):
+    h5 = "h5"
+    code = "code"
+    vcd = "vcd"
+
+
+@app.get("/result/{rid}/{result_file_type}/")
+async def get_result(rid: str, result_file_type: ResultFileType) -> FileResponse:
+    """Gets the H5 format result file of the given RID and returns it.
```

Comment:
The returning file is "H5 format result file" only if the `result_file_type` is h5, right?

Reply:
Yes😂 I didn't update the docstring..